src/app/components/calendar/calendar.ts

```diff
@@ -1268,6 +1277,40 @@ export class Calendar implements OnInit,OnDestroy,ControlValueAccessor {
         return between;
     }
 
+    isYearEquals(value, year) {
+        if (value && value instanceof Date) return value.getFullYear() === year;
+        else return false;
+    }
+
+    isYearBetween(start, end, year) {
+        let between: boolean = false;
+        if (start && start instanceof Date && end && end instanceof Date) {
+            return start.getFullYear() <= year && end.getFullYear() >= year;
+        }
+
+        return between;
+    }
+
+    isLessThanMinYear(year) {
+        if (this.minDate) {
```

Comment:
I think it's better.
```ts
return this.minDate && year < this.minDate.getFullYear();
```

Reply:
Oh, yes. You are right. I'll update it. Thanks!